TESTS/configs/baremetal.json

```diff
@@ -1,12 +1,12 @@
-{ 
-    "requires": [ 
-        "bare-metal", 
+{
```

Comment:
Can you remove the space corrections so the review focuses only on what matters? Space removal can be fixed later on. It would have been ok if it was only once or twice but you have many here.

Reply:
Sorry,I prefer not to get review comments from other reviewers on this indentation/space issue, As Evelyne previously mentioned this needs to be addressed first.

Reply:
Remove the space changes you made.